src/fileshelter/ui/FileShelterApplication.cpp

```diff
@@ -26,6 +26,8 @@
 #include <Wt/WMenu.h>
 #include <Wt/WTemplate.h>
 #include <Wt/WText.h>
+#include <Wt/WAnchor.h>
+#include <Wt/WHBoxLayout.h>
```

Comment:
Useless added includes

Reply:
Yep - forgot to tidy up - need to reapply changes to the original version when I am finished or check before I commit so it is cleaner.